test/examples/universalBrowser.test.js

```diff
@@ -1,60 +1,62 @@
-/* globals expect */
-/* globals describe */
-/* globals it */
-/* globals sinon */
+import sinon from 'sinon';
+import { expect } from 'chai';
+
+import { setTimeout } from '../util/timers.js';
+import { loadHtml } from '../util/fixtures.js';
 
 describe('Rollbar loaded by snippet', function () {
-  before(function (done) {
-    // Stub the xhr interface.
-    window.server = sinon.createFakeServer();
+  let __originalOnError = null;
+
+  before(async function () {
+    // Prevent WTR/Mocha from failing the test on uncaught errors.
+    __originalOnError = window.onerror;
+    window.onerror = () => false;
```

Comment:
We store the original error and replace it with one that just returns false to signal the exception has been handled and to prevent propagation. Then Rollbar will swap its onerror with this one instead of WTR/Mocha/React's.